src/store/modules/ProfilesModule.ts

```diff
@@ -0,0 +1,25 @@
+interface State {
+    profileList: string[];
+}
+
+/**
+ * State for Profiles, i.e. list for profiles in a single game/community.
+ */
+export default {
```

Comment:
Nitpick: another thing we've been trying to move away from is the default exports. So `export const ProfilesModule = {` would be preferred here. (IIRC Vue components require default export though.)